### PR TITLE
Feature: initial home page setup

### PR DESCRIPTION
**Pull request by Thevel0per**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,12 @@ gem 'activerecord', '7.0.3.1'
 gem 'activesupport', '7.0.3.1'
 gem 'railties', '7.0.3.1'
 
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails', '~> 3.4', '>= 3.4.2'
```
Comment (tmtocb) [inline Gemfile:16]: hardcore Sprockets-Rails version

File: Gemfile [hunk 2]
```diff
@@ -12,6 +12,12 @@ gem 'activerecord', '7.0.3.1'
...
+
+# SASS processor
+gem 'dartsass-rails', '~> 0.4.0'
```
Comment (tmtocb) [inline Gemfile:19]: hardcore DartSass-Rails version

File: Gemfile [hunk 3]
```diff
@@ -42,6 +48,7 @@ group :development do
   gem 'better_errors', '2.9.1'
   gem 'binding_of_caller', '1.0.0'
   gem 'bullet', '7.0.2'
+  gem 'foreman'
```
Comment (tmtocb) [inline Gemfile:51]: hardcore Foreman version

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+.navigation {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &__links {
+    display: flex;
+    gap: 16px;
+    align-items: center;
+  }
+}
```
Comment (tmtocb) [inline app/assets/stylesheets/application.scss:30]: add empty line at the end of the file

File: app/controllers/home_controller.rb [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
```
Comment (tmtocb) [inline app/controllers/home_controller.rb:3]: inherit from ApiController instead of ApplicationController

File: Gemfile (hunk 3 above)
Comment (tmtocb) [inline Gemfile:51]: add ticket to the Project Board 'add Foreman' gem with label Gemfile

**Review by tmtocb — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 3 above)
Comment (tmtocb) [reply Gemfile:51]: Done: https://github.com/wta-fourty-love/application/issues/338

File: .gitignore [hunk 6]
```diff
@@ -21,6 +21,10 @@
 # Multi Environment Encrypted Secure Credentials keys:
 /config/credentials/production.key
 
+# Ignore dart-sass related stuff
```
Comment (tmtocb) [inline .gitignore:24]: add ticket to the Project Board 'add DartSass assets builds context to .gitignore' with labels GitHub, CSS link it to feature branch

Comment (tmtocb) [reply .gitignore:24]: Done: https://github.com/wta-fourty-love/application/issues/339

**Please resolve when you see it**

File: bin/dev [hunk 7]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```
[the remark this answers is not in the document]
Comment (tmtocb) [reply bin/dev:2]: Done: https://github.com/wta-fourty-love/application/issues/340

**Please resolve when you see it**

**Review by tmtocb — CHANGES_REQUESTED** [review]

File: config/locales/pl.yml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+pl:
+  latest_results: "Ostatnie wyniki"
```
[the remark this answers is not in the document]
Comment (tmtocb) [reply config/locales/pl.yml:2]: Done: https://github.com/wta-fourty-love/application/issues/341

**Please resolve when you see it**

**Comment by Thevel0per** [comment]
This has been split into different tasks and features, closing.

**Pull request closed without merging**